interface-definitions/service_lldp.xml.in

```diff
@@ -29,7 +29,34 @@
               </constraint>
             </properties>
             <children>
-              #include <include/generic-disable-node.xml.i>
+              <leafNode name="mode">
+                <properties>
+                  <help>Set LLDP receive/transmit operation mode of this interface</help>
+                  <completionHelp>
+                    <list>disable rx-and-tx tx-only rx-only</list>
```

Comment:
I wonder if just `tx`, `rx`, and `tx-rx` would suffice.

Reply:
I opted for rx-tx to keep the alphabetical order.